Language_Parser/actionVerbs.py

```diff
@@ -325,17 +325,284 @@ def listen(info):
     print(room.verbResponses("Listen", item))
 
 
+def removeOldItem(player, item, location):
+    """
+    Helper function for 'use' and combining objects -
+    removes an Item instance from the specified location.
+    Location should be one of these strings: "inv", "roomVis", "roomDrop"
+    """
+    if location == "inv":
+        player.removeInventory(item)
+    elif location == "roomVis" or location == "roomDrop":
+        player.getLocation().removeAccessibleItem(item)
+
+
+def placeNewItem(player, item, location):
+    """
+    Helper function for 'use' and combining objects -
+    places an instance of the Item in the specified location.
+    Location should be one of these strings: "inv", "roomVis", "roomDrop"
+    """
+    if location == "inv":
+        player.addInventory(item)
+    elif location == "roomVis":
+        player.getLocation().addVisibleItem(item)
+    elif location == "roomDrop":
+        player.getLocation().addVisibleItem(item)
+
+
+def combineItemAndFeature(player, item, feature):
+    """
+    Helper function for 'useHandler'.
+    Takes player and game data, and two strings of the item and feature names.
+    Returns None if the combination fails.
+    """
+    # Since order of words doesn't matter, choose an order for processing
+    if {item, feature} == {"catFood", "dish"}:
+        item = "catFood"
+        feature = "dish"
+        itemData = getItemDataForUse(player, item)
+    else:
+        itemData = []
+
+    currentRoom = player.getLocation()
+
+    # Look up item interaction response, if there is one
+    try:
+        response = currentRoom.verbResponses("Use", item, feature)
+        # If that doesn't get a valid response, try swapping the inputs
+        if response == "None":
+            response = currentRoom.verbResponses("Use", feature, item)
+    except (KeyError, ValueError, IndexError):
+        response = None
+
+    if response != "None":
+        # Case for "catFood", "dish"
+        if {item, feature} == {"catFood", "dish"} and currentRoom == "porch":
+            # Remove catFood from inventory
+            removeOldItem(player, itemData[1], itemData[2])
+            # Call 'use' description
+
```

Comment:
Later it will, yes. There are quite a few things I can't test until more of the conditionals are done. But I can make this a TODO to make sure it's not forgotten.